CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## 3.0.7
 
 ### Changes
-
+- Add AddSentryTag and AddSentryContext Extensions for exception class (#834) @lucas-zimerman
```

Comment:
Oh I just made a release, didn't see this PR got updated.

Can you please move this to the top:

```

## Unpublished

### Features

- Add AddSentryTag and AddSentryContext Extensions to Exception (#834) @lucas-zimerman


```

Reply:
@bruno-garcia  done